client/src/pages/ProjectsPage.js

```diff
@@ -4,18 +4,42 @@ import Wrapper from "../components/Wrapper";
 import Footer from "../components/Footer";
 
 class Projects extends Component {
+
+    state = {
+        title: "",
+        description: "",
+        image: ""
+    };
+
+    handleSaveProject = event => {
+        alert("Project Saved!");
+        // handle saving project here
+    }
+
+    handleSubmit = event => {
+        this.setState({title: event.target.value})
+        this.setState({description: event.target.value})
+        this.setState({image: event.target.value})
+    }
+
+
     render() {
         return (
             <div>
                 <HomeNav />
                 <Wrapper>
                     <div>
                         <h1 className="subTitle">Projects</h1>
-                    </div>
+                    </div><br/>
```

Comment:
`<br>`s have a tendency to be frowned upon since you can achieve line breaks from other css techniques (clear: both, flex, grid, etc). Not a large issue, worth considering.